src/modules/import/api/import.controller.ts

```diff
@@ -170,4 +182,40 @@ export class ImportController {
             }
         }
     }
+
+    @Get('history')
+    @ApiOperation({ description: 'Get the history of import.' })
+    @ApiOkResponse({
+        description: 'The import transactions were successfully returned',
+        type: [ImportVorgangResponse],
+        headers: PagingHeadersObject,
+    })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to get import transactions.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to get import transactions.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while getting import transactions.' })
```

Comment:
Since this is an endpoint that is related to the admin UI, it should probably include `@ UseGuards(StepUpGuard)`.

Reply:
Okay, thanks